src/test/java/org/symphonyoss/symphony/messageml/elements/UIActionTest.java

```diff
@@ -345,15 +424,15 @@ public void testUIActionOpenDialogInDifferentScopeFromDialog() throws Exception
             + "<title>title</title>"
             + "<body>body</body>"
             + "</dialog>"
-            + "<form>"
+            + "<form id=\"form-id\">"
             + "<ui-action trigger=\"click\" action=\"open-dialog\" target-id=\"target-dialog-id\">"
             + "<button>Open the dialog</button>"
             + "</ui-action>"
             + "</form>"
             + "</messageML>";
 
     expectedException.expect(InvalidInputException.class);
-    expectedException.expectMessage("ui-action with a target-id must have only one dialog sibling with a matching id");
+    expectedException.expectMessage("The form with id 'form-id' should have at least one action button");
```

Comment:
Just to double check?
Does this appear every time that a form has an `id` ? Or only in the dialog context ?

Reply:
There was actually a mistake in the test, I added the button to the form and now we are back to the first message error :)